src/Dynamo.All.sln

```diff
@@ -247,6 +247,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tools", "Tools", "{C963F7A9
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NodeDocumentationMarkdownGeneratorTests", "..\test\Tools\NodeDocumentationMarkdownGeneratorTests\NodeDocumentationMarkdownGeneratorTests.csproj", "{4C2AEACC-617D-4B90-ADCD-FE294A43A68F}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "IronPythonTests", "..\test\Libraries\IronPythonTests\IronPythonTests.csproj", "{E6DF2FBD-7D4D-4465-94DC-D576D737E985}"
```

Comment:
Why is this being added instead of removing it from the solution? Is it simply added but not participating in the build?